usaspending_api/api_contracts/contracts/v2/federal_accounts/federal_account_id/spending_over_time.md

```diff
@@ -0,0 +1,96 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# List of Available Program Activities [/api/v2/federal_accounts/{federal_account_id}/spending_over_time]
```

Comment:
Looks like this was copy-pasted from the Program Activity endpoing. Suggested heading/description - `Account Spending by Time Period` / `This endpoint supports the Federal Account page and allows for listing spending by the Federal Account within certain periods of time.`